sdk/communication/communication-common/src/identifierModels.ts

```diff
@@ -59,6 +60,31 @@ export interface MicrosoftTeamsUserIdentifier {
   cloud?: "public" | "dod" | "gcch";
 }
 
+/**
+ * A Microsoft Teams bot.
+ */
+export interface MicrosoftTeamsBotIdentifier {
+  /**
+   * Optional raw id of the Microsoft Teams bot.
+   */
+  rawId?: string;
+
+  /**
+   * Id of the Microsoft Teams bot.
```

Comment:
```suggestion
   * The unique Microsoft app ID for the bot as registered with the Bot Framework. 
```

as in https://learn.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema#bots

Reply:
Done